### PR TITLE
Add  Image form handler servlet and modify pom.xml

### PR DESCRIPTION
**Pull request by SantiM1**
Add the function to upload images to cloud storage and modify pom.xml to add cloudstorage dependencies and set the correct project id

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -38,6 +38,12 @@
       <artifactId>jetty-annotations</artifactId>
       <version>${jetty.version}</version>
     </dependency>
+     <!-- Google Cloud Storage -->
```
Comment (Renaud2002) [inline pom.xml:41]: @SantiM1 funny thing is I add the google cloud storage dependency to my pom.xml too we might run into a merge conflict, if we do I can fix it no worries.

**Review by Renaud2002 — APPROVED** [review]
Looks good, great job.

File: pom.xml (hunk 1 above)
Comment (SantiM1) [reply pom.xml:41]: Yes the conflict should be pretty easy to spot, the code is not very complex

**Review by erivas28 — COMMENTED** [review]
Looks great so far Santi

**Review by erivas28 — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (Renaud2002) [reply pom.xml:41]: Agree

**Review by Renaud2002 — APPROVED** [review]
Looks good my man.

**Pull request merged**